modules/ROOT/pages/provisioning-qemu.adoc

```diff
@@ -38,9 +38,15 @@ If you need to persist your changes, it is recommended to set up a dedicated per
 .Launching FCOS with QEMU (temporary storage)
 [source, bash]
 ----
+# for x86/aarch64:
+IGNITION_DEVICE="fw_cfg name=opt/com.coreos/config,file=path/to/example.ign"
+
+# for s390x/ppc64le:
+IGNITION_DEVICE="drive file=path/to/example.ign,if=none,format=raw,readonly=on,id=ignition -device virtio-blk,serial=ignition,drive=ignition"
+
 qemu-kvm -m 2048 -cpu host -nographic -snapshot \
 	-drive if=virtio,file=fedora-coreos-qemu.qcow2 \
-	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign \
+	-${IGNITION_DEVICE} \
 	-nic user,model=virtio,hostfwd=tcp::2222-:22
```

Comment:
Let's take an opportunity here to break out a few more things into variables too.

i.e. let's use `IGNITION_CONFIG` and `IMAGE` variables like is done in the libvirt directions.

Reply:
addressed in eab3b9da4c5293a514974465205b2a8bfe8e28e7